src/excalidraw-app/index.tsx

```diff
@@ -605,7 +653,6 @@ const ExcalidrawWrapper = () => {
         ref={excalidrawRefCallback}
         onChange={onChange}
         initialData={initialStatePromiseRef.current.promise}
-        onCollabButtonClick={() => setCollabDialogShown(true)}
```

Comment:
We still need this to show the collab button next to the library button. Though I'd remove this in a future PR as we should support rendering that button(s) from host.